sys-utils/lscpu-arm.c

```diff
@@ -70,9 +70,12 @@ static const struct id_part arm_part[] = {
     { 0xd08, "Cortex-A72" },
     { 0xd09, "Cortex-A73" },
     { 0xd0a, "Cortex-A75" },
+    { 0xd0b, "Cortex-A76" },
+    { 0xd0c, "Neoverse-N1" },
```

Comment:
Should we include on top of this struct cross reference comment pointing to https://github.com/gcc-mirror/gcc/blob/master/gcc/config/aarch64/aarch64-cores.def

Reply:
I'm not sure that is complete, plus it appears some of those are more "gccisms" or unreleased cores. 

The first party docs might be:
Cortex-A76
https://developer.arm.com/docs/100798/0300/part-b-register-descriptions/aarch64-system-registers/midr_el1-main-id-register-el1

Similarly:
Neoverse-N1
https://developer.arm.com/docs/100616/0301
Neoverse-E1
https://developer.arm.com/docs/101560/0101

Reply:
Above reference links are really good in commit message that adds specific models. Such a shame arm does not have (or makes it difficult to find) a list of all models. The closest I could find was that gcc file.